### ISSUE TITLE
Ensure header links are accessible

### ISSUE THREAD

**Issue by mcquinne** [cires-ncei/onestop#77]
The links in the header (home, about, etc.) should be accessible via screen reader.

Also we should ensure that there's a screen reader accessible way to dismiss the dialogs.

**Comment by mcquinne** [comment]
FYI, based on [this tutorial](https://www.w3.org/WAI/tutorials/) I discovered that there's a `role="alert"` attribute that you can put on elements to indicate to screen readers that they are alerts that should be read to the user. So probably we should make sure our "Info" and "Help" blocks have that attribute when added to the page, like:

```html
<div role="alert">
  <p>Helpful help text</p>
</div>
```

Or maybe it should go on the paragraph tag? I'm not sure.

**Comment by mcquinne** [comment]
Also, we should wait until #43 is done to do this.

**Comment by aaronjcaldwell** [comment]
It's fairly easy to add the alert tag to elements however I don't think it fits for the 'About' and 'Help' elements based upon what I'm seeing [here](https://www.w3.org/TR/wai-aria/roles#alert).

**Comment by ajakz** [comment]
This might be a helpful example: http://oaa-accessibility.org/example/21/

**Comment by ajakz** [comment]
Header navigation links -- normal menu and burger style -- are read by screen reader & grouped in a sensical fashion. Clicking them via VoiceOver works as expected. However, two issues remain: #85 to read the Help & About panels; and clicking a link doesn't really inform the screen reader when a new page has loaded. I'll make note of the 2nd issue on the 508 project board. Otherwise, I would consider the limited scope of this story to be complete.
